enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/statemachine/service/StateMachineService.java

```diff
@@ -123,6 +134,33 @@ public Message<OnboardingEvent> createMessage(OwnerId ownerId, String processId,
                 .build();
     }
 
+    /**
+     *
+     */
+    @Transactional(readOnly = true)
+    public void changeMachineStatesInBatch() {
+        try (Stream<IdentityVerificationEntity> stream = identityVerificationService.streamAllIdentityVerificationsToChangeState().parallel()) {
+            stream.forEach(this::changeMachineState);
+        }
+    }
+
+    private void changeMachineState(final IdentityVerificationEntity identityVerification) {
+        final String processId = identityVerification.getProcessId();
+        final OwnerId ownerId = new OwnerId();
+        ownerId.setActivationId(identityVerification.getActivationId());
+        ownerId.setUserId(identityVerification.getUserId());
```

Comment:
👀 Needed to have the real `userId` from `IdentityVerificationEntity` especially for the presence check provider.